### PR TITLE
Use SGD hyperparameters that are more stable for initial value tests.

### PR DESCRIPTION
**Pull request by dickensc**
Use SGD hyperparameters that are more stable for initial value tests.

For smaller models, like the one used in PSL tests, a slower decay of the learning rate leads to more consistent results.

### CODE REVIEW

**Comment by eriq-augustine** [comment]
lgtm
